### PR TITLE
removed the likeToBookmarkEx

### PR DESCRIPTION
**Pull request by smclaug7**

### CODE REVIEW

File: fs-gallery-view-selector.html [hunk 1]
```diff
@@ -289,22 +289,20 @@
...
-          <h5 class='albums-divider'>[[_fetchMyFavoritesText(i18n, _likeToBookmarkEx)]]</h5>
+          <h5 class='albums-divider'>[[_fetchMyFavoritesText(i18n)]]</h5>
         </li>
-        <li class='items-view' data-uid='my-favorites' on-dragover='_handleDrag' on-dragleave='_clearDrag' on-drop='_handleDrop' title$="[[_fetchLabelHoverText(i18n, _likeToBookmarkEx)]]">
+        <li class='items-view' data-uid='my-favorites' on-dragover='_handleDrag' on-dragleave='_clearDrag' on-drop='_handleDrop' title$="[[_fetchLabelHoverText(i18n)]]">
           <div>
-            <i class='album-icon my-favorites-icon' hidden='[[_likeToBookmarkEx]]'></i>
-            <i class='album-icon my-bookmarks-icon' hidden='[[!_likeToBookmarkEx]]'></i>
-            <span data-test="MyLikesFolder">[[_fetchFavoriteMemoriesText(i18n, _likeToBookmarkEx)]]</span>
+            <i class='album-icon my-favorites-icon'></i>
```
Comment (dcravenus) [inline fs-gallery-view-selector.html:296]:
```suggestion
            <i class='album-icon my-bookmarks-icon'></i>
```

Comment (dcravenus) [reply fs-gallery-view-selector.html:296]: I believe we want the bookmarks icon to show and the favorites icon to hide permanently

File: fs-gallery-view-selector.html [hunk 2]
```diff
@@ -289,22 +289,20 @@
...
             <span class='fs-badge' data-test="MyLikesCount" hidden$='[[_hideCount(hideCounts, view, "my-favorites")]]'>[[_getCount(favoritesCount, hideCounts, count)]]</span>
           </div>
         </li>
         <!-- Generated list of liked albums -->
         <template id="album-list" is='dom-repeat' items='{{likedAlbums}}' as="likedAlbum" sort='_sortByName'>
           <li class='items-view album-item' data-uid$='[[likedAlbum.id]]' title$="[[likedAlbum.name]]">
             <div>
-              <i class='album-icon liked-folder-icon' hidden='[[_likeToBookmarkEx]]'></i>
-              <i class='album-icon bookmarked-folder-icon' hidden='[[!_likeToBookmarkEx]]'></i>
+              <i class='album-icon liked-folder-icon'></i>
```
Comment (dcravenus) [inline fs-gallery-view-selector.html:305]:
```suggestion
              <i class='album-icon bookmarked-folder-icon'></i>
```

Comment (dcravenus) [reply fs-gallery-view-selector.html:305]: I believe we want the bookmarked icon to show and the liked icon to hide permanently

**Comment by dcravenus** [comment]
If the inlined svgs in the CSS for "like" icons aren't being used anymore we can delete those as well.

File: fs-gallery-view-selector.html (hunk 1 above)
Comment (smclaug7) [reply fs-gallery-view-selector.html:296]: Yeah, I realized that just now another file had the same type of logic and I realized I switched it in my brain.

File: fs-gallery-view-selector.html (hunk 2 above)
Comment (smclaug7) [reply fs-gallery-view-selector.html]: Fixed

**Review by dcravenus — APPROVED** [review]

**Pull request merged**